python/pyarrow/tests/test_ipc.py

```diff
@@ -329,6 +329,40 @@ def test_stream_simple_roundtrip(stream_fixture, use_legacy_ipc_format):
         reader.read_next_batch()
 
 
+@pytest.mark.zstd
+def test_compression_roundtrip():
+    if not pa.Codec.is_available('zstd'):
+        pytest.skip("{} support is not built".format('zstd'))
```

Comment:
Sorry, but - is this check redundant with the mark?

Reply:
Don't be sorry :).  Thanks for catching it.  I've pushed a fix.